docs/reference/bundles/payment/ogone.rst

```diff
@@ -65,5 +65,5 @@ Here's how to configure Ogone for Sonata e-commerce:
             # ...
 
             # If you've overridden the OgonePayment class
-            sonata.payment.method.ogone.class:      "Application\\Sonata\\Component\\Payment\\Ogone\\OgonePayment"
-            #...
\ No newline at end of file
+            sonata.payment.method.ogone.class:      "App\Application\\Sonata\\Component\\Payment\\Ogone\\OgonePayment"
```

Comment:
According to the string, I think there is a second backslash missing after "App".

Reply:
True